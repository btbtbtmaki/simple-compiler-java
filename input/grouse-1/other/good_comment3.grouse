// comment starting a file
main {
	imm c := 3 + 4 * 5;  
	imm d := 6 * 5;
	imm e := 7 * 5;
	print c, d, e, nl;
}
