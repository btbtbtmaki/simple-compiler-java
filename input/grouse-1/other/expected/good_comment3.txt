23 30 35 
